docs/misc/changelog.rst

```diff
@@ -27,6 +27,8 @@ New Features:
 - Wrap the environments automatically with a ``Monitor`` wrapper when possible.
 - ``EvalCallback`` now logs the success rate when available (``is_success`` must be present in the info dict)
 - Added new wrappers to log images and matplotlib figures to tensorboard. (@zampanteymedio)
+- Add support for text records to ``Logger``. (@lorenz-h)
+- Add hint on how to access the tensorboard summary writer directly to docs. (@lorenz-h)
```

Comment:
the second entry should go under documentation ;)

Reply:
Adressed in ffacc5f8af4c77cfea7b1b9c24ecbd7bb3bbbc4d